relayer/relays/parachain/beefy-listener.go

```diff
@@ -234,9 +234,8 @@ func (li *BeefyListener) generateProof(ctx context.Context, input *ProofInput, h
 
 	// Generate the MMR proof for the polkadot block.
 	mmrProof, err := li.relaychainConn.GenerateProofForBlock(
-		input.RelayBlockNumber+1,
+		input.RelayBlockNumber,
```

Comment:
Ah, nope, I explicitly had to use "+ 1" here in the previous PR (refactor OutboundQueue) to get the right proofs. This is different from the beefy-relayer.

You can test BridgeHub->Ethereum messaging by calling the `EthereumControl.upgrade` extrinsic in the parachain .